Week_01_Basics/Assignment/portfolio.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+  <head>
+    <meta charset="UTF-8" />
+    <link href="./style.css" rel="stylesheet" type="text/css" />
+    <title>Portfolio - Joe Johnson</title>
+  </head>
+  <body>
+    <div class="main">
+      <br />
+      <div id="sitemap"> 
+        <div id="hlink"><a href="./about.html">About Me</a></div>&nbsp;|&nbsp;
+        <div id="onPage">Portfolio</div>
+      </div>
+      <h1>JOE JOHNSON</h1>
+      <hr />
+    
+      <h2>Experience</h2>		
+      <p>I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an <b><i>awesome</i></b> Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.</p>
+      <h3>PORTFOLIO</h3>
+      <!--RESUME PAGE-->
+      <ul>
+        <li>
+          <div id="position">Web Developer,</div>&nbsp;
+          <div id="company">Relaxr</div>&nbsp;
+          <div id="period">2015 - present</div>
+          <div id="desc">Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.</div>
+          <p><img src="./images/relaxr_deliverable.png" id="portfolioImage"/></p>
+        </li>
+        <li>
+          <div id="position">Web Developer,</div>&nbsp;
```

Comment:
You can use ```<span>``` instead of div here.  It's like div, but already has the ```display: inline``` style, and it's used changing individual texts inside a html element.